sip-ri/src/main/java/gov/nist/core/NameValueList.java

```diff
@@ -416,4 +436,10 @@ protected Map<String,NameValue> getMap() {
 		}
 		return hmap;
 	}
+
+    
+   // Set the flag indicating whether backslashes are treated as escape characters @MukhaValeriia
+	public void setEscaped(boolean b) {
+		
```

Comment:
This method is empty. Is it really needed ?

Reply:
Yes, my bad. Removed this method in TargetDialog.java in previous commit and didn't update here